### PR TITLE
Feature/eslint

### PR DESCRIPTION
**Pull request by kevgathuku**
Switch from jshint to eslint for linting
@andela-gukpere please review

### CODE REVIEW

File: app/scripts/components/login/loginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+var React = require('react'),
+  UserActions = require('../../actions/userActions'),
+	UserStore = require('../../stores/userStore'),
+	Navigation = require('react-router').Navigation;
+
+var LoginForm = React.createClass({
+	mixins: [Navigation],
+	
+	getInitialState: function() {
```
Comment (andela-gukpere) [inline app/scripts/components/login/loginForm.jsx]: **Code Quality - Nitpick**

Debug and configure your text editor so your spacing and indentation is consistent across `.jsx` and `.js` scripts

File: server/controllers/users.js [hunk 2]
```diff
@@ -23,18 +23,58 @@
     login: function(req, res, next) {
       passport.authenticate('login', function(err, user) {
         if (err) {
-          return res.status(500).send(err);
+          return res.send(err);
         }
         if (!user) {
           return res.status(500).send({
```
Comment (andela-gukpere) [inline server/controllers/users.js]: **Code Quality - Nitpick**

Status code `500` refers to an undetermined error on the server. In this case, and invalid login should use 401 which means _*Not Authorized*_, as we know exactly what the error is.

File: server/config/db-connect.js [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+
+  var Sequelize = require('sequelize'),
+    env = process.env.NODE_ENV || 'development',
+    config = require('./index')[env],
+    sequelize = new Sequelize(
+      config.db.name,
+      config.db.username,
+      config.db.password, {
+        host: config.host,
+        dialect: config.db.dialect || 'postgres'
```
Comment (andela-gukpere) [inline server/config/db-connect.js]: **Code Quality - Nitpick**

The derivation of configuration values should be done in the config script.
The _or_(ing) of the dialect and other configurations should be moved to the config script.

File: app/shared/head.jade [hunk 4]
```diff
@@ -4,7 +4,10 @@ meta(http-equiv="X-UA-Compatible" content="IE=edge")
 meta(name="viewport" content="width=device-width initial-scale=1")
 meta(name="description" content="")
 title Trime
-link(rel="stylesheet" href="css/style.css")
-link(rel='stylesheet', href='https://storage.googleapis.com/code.getmdl.io/1.0.6/material.cyan-teal.min.css')
-script(src='https://storage.googleapis.com/code.getmdl.io/1.0.6/material.min.js')
+link(rel='stylesheet', href='https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.3/css/materialize.min.css')
 link(rel='stylesheet', href='https://fonts.googleapis.com/icon?family=Material+Icons')
+script(type='text/javascript', src='https://code.jquery.com/jquery-2.1.1.min.js')
```
Comment (andela-gukpere) [inline app/shared/head.jade]: Do we use jQuery in our application?

File: app/scripts/components/login/signupForm.jsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+              <hr />
+		          <SignupForm />
+		        </div>
+		      </div>	
+    		</div>
+			);
+		}
+	});
+
+module.exports = SignupCard;
```
Comment (andela-gukpere) [inline app/scripts/components/login/signupForm.jsx]: **Code Quality - Nitpick**

We should refactor the variable so we export the `SignupForm` as the script name depicts.

File: app/shared/head.jade (hunk 4 above)
Comment (kevgathuku) [inline app/shared/head.jade]: @andela-gukpere yes, jQuery is required by materialize JS

**Comment by kevgathuku** [comment]
@andela-gukpere All suggested changes have been made in the `feature/login` branch, which also contains all the changes made in this branch.

**Pull request closed without merging**